eth/eventdatahandler/task_executor_test.go

```diff
@@ -5,12 +5,13 @@ import (
 	"testing"
 
 	spectypes "github.com/bloxapp/ssv-spec/types"
-	ethtypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/stretchr/testify/require"
-	"go.uber.org/zap/zaptest"
-
 	"github.com/bloxapp/ssv/eth/eventbatcher"
 	ssvtypes "github.com/bloxapp/ssv/protocol/v2/types"
+	ethcommon "github.com/ethereum/go-ethereum/common"
```

Comment:
Let's agree on a import sorting rule to avoid import reformatting after each other. I'd suggest having 3 blocks:

1) standard library
2) 3rd party dependencies
3) project dependencies

However, I'm not sure how to setup it this way in VSCode. Ignore this comment if it's complex to set up